### PR TITLE
[CWS] add non seccomp support

### PR DESCRIPTION
**Pull request by safchain**
### What does this PR do?

This PR allows to start the ptracer without the seccomp mode

### CODE REVIEW

**Review by spikat — APPROVED** [review]

File: pkg/security/ptracer/ptracer.go [hunk 1]
```diff
@@ -239,17 +240,110 @@ func (t *Tracer) ReadArgStringArray(pid int, regs syscall.PtraceRegs, arg int) (
...
+			case syscall.PTRACE_EVENT_CLONE, syscall.PTRACE_EVENT_FORK, syscall.PTRACE_EVENT_VFORK:
+				if npid, err := syscall.PtraceGetEventMsg(pid); err == nil {
+					cb(CallbackPostType, nr, int(npid), pid, regs, nil)
+				}
+			case syscall.PTRACE_EVENT_EXEC:
+				cb(CallbackPostType, ExecveNr, pid, 0, regs, nil)
+
+				if state := tracker.PeekState(pid); state != nil {
+					state.Exec = true
+				}
```
Comment (spikat) [inline pkg/security/ptracer/ptracer.go:310]: I'm not sure it's possible, but if the FIRST syscall a prog make is an exec (like, directly after the exec launching it), the state won't be initialized yet here (by NextStop()) and PeekState will return nil

**Review by spikat — APPROVED** [review]
Added a comment for a possible issue, otherwise LGTM :+1:

File: pkg/security/ptracer/ptracer.go (hunk 1 above)
Comment (safchain) [reply pkg/security/ptracer/ptracer.go:310]: here that is post of an exec, the entry is line 316, so we should have already a NextStop for this pid. I'm not sure that the functional tests will pass if it wasn't the case. Did I miss something regarding your comment ?

Comment (spikat) [reply pkg/security/ptracer/ptracer.go:310]: You're right, I misread this part :)

**Review by paulcacheux — APPROVED** [review]

**Review by chouetz — APPROVED** [review]

**Pull request merged**